.github/workflows/sync-release-branch.yaml

```
@@ -0,0 +1,30 @@
+name: Sync release branch to master
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - release-*
+
+jobs:
+  sync-branches:
+    runs-on: ubuntu-latest
+    name: Syncing release branch to master
+    outputs:
+      pr_number: ${{ steps.create_pr.outputs.pr_number }}
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Set up Node
+      uses: actions/setup-node@v1
```

```suggestion
      uses: actions/setup-node@v3
```
also to align with `dapr-test-sdk.yml`